### DISCUSSION TITLE
Global Campus

### DISCUSSION THREAD

**Discussion by JennPlothow** [community/community#50399] [GitHub Education]
![crachauninove](https://user-images.githubusercontent.com/81450408/226135004-848d98d3-0835-4acb-a27b-c9391149159c.png)
OLÁ COMO INGRESSAGAR PARA A Global Campus?

**Comment by samus-aran** [comment]
Hi @JenniferPlothow 👋 

We would recommend you check out the [Student Developer Pack FAQs](https://github.com/orgs/community/discussions/17814) for tips on how to apply.

Once you become a verified student you have access to [Global Campus](https://education.github.com/globalcampus/student). 

GitHub Global Campus is a portal from which you can access your GitHub Education benefits and resources, all in one place. On the GitHub Global Campus portal, students can:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

It's great to have you in the GitHub Education community and we hope to see you around 👋
